articles/flask-database-integration-with-sqlalchemy/index.md

```diff
@@ -0,0 +1,329 @@
+# Flask Database Integration With SQLAlchemy
+Databases are integral components of building web applications. Throughout the lifecycle of a web application, the user sends bits of data that are needed to be stored for future reference purposes. Simultaneously, the user also requests information from where they are stored.
+
+Data and Information are facets that make web applications valuable. If the user can’t send and receive data from your web application, then it’s not providing value in its usage. Therefore, the information supplied by the user has to be stored in a database so it can be accessed later.
+
+In Flask web applications, to manipulate databases, we will be using SQL and Object Relational Mapping(ORM). An ORM makes writing SQL queries easier for a programmer, because it enables us to write queries in an object-oriented language, and then ORM automatically translates it to SQL and retrieves the result in the form of an object.
```

Comment:
```suggestion
In Flask web applications, to manipulate databases, we will be using SQL and Object Relational Mapping (ORM). An ORM makes writing SQL queries easier for a programmer, because it enables us to write queries in an object-oriented language, and then ORM automatically translates it to SQL and retrieves the result in the form of an object.
```